src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -1259,6 +1259,53 @@ private void ImportBinaryOperation(ILOpcode opcode)
                     {
                         _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ThrowDivZero), "_divbyzero");
                     }
+
+                    _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.DblRem), "rem");
+                    _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.FltRem), "rem");
+                    break;
+            }
+        }
+
+        private void ImportConvert(WellKnownType wellKnownType, bool checkOverflow, bool unsigned)
+        {
+            switch (wellKnownType)
+            {
+                case WellKnownType.Byte:
+                case WellKnownType.UInt16:
+                case WellKnownType.SByte:
+                case WellKnownType.Int16:
+                case WellKnownType.Int32:
+                    if (checkOverflow)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Dbl2IntOvf), "conv_i4_ovf");
+                    }
+                    break;
+                case WellKnownType.Int64:
+                    if (checkOverflow)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Dbl2LngOvf), "conv_i8_ovf");
+                    }
+                    break;
+                case WellKnownType.UInt32:
+                    if (checkOverflow)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Dbl2UIntOvf), "conv_u4_ovf");
+                    }
+                    break;
+                case WellKnownType.UInt64:
+                    if (checkOverflow)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Dbl2ULngOvf), "conv_u8_ovf");
+                    }
+                    else
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Dbl2ULng), "conv_u8");
+                    }
+                    break;
+                case WellKnownType.Single:
+                case WellKnownType.Double:
+                    _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Lng2Dbl), "conv_r");
+                    _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ULng2Dbl), "conv_r");
                     break;
             }
```

Comment:
```suggestion
            }
            if (checkOverflow)
            {
                _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Overflow), "_ovf");
            }
```
Looks like pre-existing bug